hackerrank/test/Algorithms/Warmup/CompareTriplets_ResultTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Fedotkin.Dotnet.HackerRank.Algorithms.Warmup.CompareTriplets;
+
+namespace Fedotkin.Dotnet.HackerRank.Tests.Algorithms.Warmup;
+
+public class CompareTriplets_ResultTests
+{
+    [Fact]
+    public void CompareTriplets_Example_AliceAndBobRates()
```

Comment:
Трэба факталогію перарабіць у тэорыю, каб замест трох быў адзін метад
```suggestion
    [Theory]
    [InlineData(/* a */ 1, 2, 3, 1, /* b */ 3, 2, 1, 1)] // Example
    [InlineData(/* a */ 5, 6, 7, 1, /* b */ 3, 6, 10, 1)] // Sample0
    [InlineData(/* a */ 17, 28, 30, 2, /* b */ 99, 16, 8, 1)] // Sample1
    public void CompareTriplets_AliceAndBobRatesAreCorrect(int a1, int a2, int a3, int aExpected, int b1, int b2, int b3, int bExpected)
```
Рэалізацыю тэставага метада перапішаш ужо сам.
Два ніжніх метады ўжо непатрэбныя: выдаліць!